docs/user/04-metrics.md

```diff
@@ -550,18 +550,20 @@ Up to three MetricPipeline resources at a time are supported.
 
 ### No Metrics Arrive at the Backend
 
-Cause: Incorrect backend endpoint configuration (e.g., using the wrong authentication credentials) or the backend being unreachable.
+Symptom: Incorrect backend endpoint configuration (e.g., using the wrong authentication credentials) or the backend being unreachable.
 
 Remedy: 
 - Check the `telemetry-metric-gateway` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`.
 - Check if the backend is up and reachable.
 
 ### Not All Metrics Arrive at the Backend
 
-Cause: The backend is reachable and the connection is properly configured, but some metrics are refused.
+Symptom: The backend is reachable and the connection is properly configured, but some metrics are refused.
+
+Cause: It can happen due to a variety of reasons. For example, a possible reason may be that the backend is limiting the ingestion rate.
 
 Remedy:
-- Check the `telemetry-metric-gateway` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`. If backend is refusing metrics limiting the rate, try the options desribed in [Gateway Buffer Filling Up](#gateway-buffer-filling-up)
+- Check the `telemetry-metric-gateway` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`. If backend is limiting the rate by refusing metrics, try the options desribed in [Gateway Buffer Filling Up](#gateway-buffer-filling-up)
```

Comment:
```suggestion
1. Check the `telemetry-metric-gateway` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`. 
2. If backend is limiting the rate by refusing metrics, try the options desribed in [Gateway Buffer Filling Up](#gateway-buffer-filling-up).
3. Otherwise, take the actions appropriate to the cause indicated in the logs.
```

Do we really want that they only check the Pod logs? Not the backend logs?